src/pages/Dashboard/components/Calendar/CalendarContainer.tsx

```diff
@@ -1,28 +1,19 @@
 import React, { useState, useEffect } from 'react';
 import moment, { Moment } from 'moment';
 
+import CalendarBody from './CalendarBody';
 import CalendarDailyTasksModal from './CalendarDailyTasksModal';
+import CalendarHeader from './CalendarHeader';
 
-import { TaskAssignation } from '../types';
-
-import {
-  CalendarContainer,
-  CalendarHeader,
-  Body,
-  CalendarHeaderPrev,
-  CalendarHeaderNext,
-  Week,
-  Day,
-  DayDiv,
-  DayDivTask,
-  DayNames,
-} from '../styles/calendar';
+import { TaskAssignation } from '../../../../types';
+
+import { Container } from '../../../../styles/calendar';
 
 interface CalendarProps {
   assignations: TaskAssignation[];
```

Comment:
Could you please change to assignments as it best describe the tasks that are assigned ?

Reply:
ok